app/views/shared/_one_time_code_input.html.erb

```diff
@@ -31,6 +31,13 @@ classes << local_assigns.delete(:class) if local_assigns[:class]
   class: classes,
 ) %>
 
+<%= text_field_tag(
+  name,
+  nil,
+  type: 'hidden',
```

Comment:
Could we use [`hidden_field_tag`](https://api.rubyonrails.org/v5.1.7/classes/ActionView/Helpers/FormTagHelper.html#method-i-hidden_field_tag), since we're not wanting a text field?

```suggestion
<%= hidden_field_tag(
  name,
  nil,
```